string[] labels
int32 index
bool head
bool base
int32 finger
---
int32 nb_label
---
#feedback
int32 feedback
